test/integration/nginx/nginx_integration_test.sh

```diff
@@ -248,3 +248,37 @@ then
   echo "${DIFF}"
   exit 1
 fi
+
+reset_test
+echo "======== Test 7: Check that extracting a child span without X-Datadog-Tags doesn't produce an error ========"
+# See <https://github.com/DataDog/dd-opentracing-cpp/issues/241>.
+
+wiremock --port 8080 >/dev/null 2>&1 & wait_for_port 8080
+curl -s -X POST --data '{ "priority":10, "request": { "method": "ANY", "urlPattern": ".*" }, "response": { "status": 200, "body": "Hello World" }}' http://localhost:8080/__admin/mappings/new
+
+echo '{
+  "service": "nginx",
+  "operation_name_override": "nginx.handle"
+}' > ${TRACER_CONF_PATH}
+
+run_nginx
+
+curl -s \
+  -H 'X-Datadog-Trace-Id: 1234' \
+  -H 'X-Datadog-Parent-Id: 1234' \
+  -H 'X-Datadog-Sampling-Priority: 0' \
+  http://localhost/proxy/ >/tmp/curl_log.txt
+
+if [ "$(cat ${NGINX_ERROR_LOG} | grep 'no opentracing context value found for span context key ' | wc -l)" != "0" ]
```

Comment:
Is there an easy way to verify that this captures any future propagation headers that would emit this error?